tests/ci/files/ostree_key

```diff
@@ -0,0 +1,38 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
We are aware that this is a private key in an open source project. Right?

Reply:
Yes. Actualy, the InfoSec team detected and ask me about it. We were talking and it wasn't really a risk, as we use them just for the temporary VM, but we agree on trying to generate one new key on the fly each run.

That's one of the few minor changes I wanted to do when the mayor changes get validated.